fellowship/contract_renderer.py

```diff
@@ -0,0 +1,43 @@
+import json
+import glob
+import os
+
+from jinja2 import Environment, FileSystemLoader
+
+from .utilis import load_config, validate_contract
+from .contract import Contract
+
+
+class ContractRenderer:
+
+    def __init__(self, template_directory: str) -> None:
+        self.config = load_config()
+        self.jinja_env = Environment(loader=FileSystemLoader(template_directory))
+        self.jinja_env.filters['jsonify'] = json.dumps

```

Comment:
Please document this filter even though it is a bit self evident. But it would be good to know for the user that there is this type of the filter available.